packages/@aws-cdk/toolkit/package.json

```diff
@@ -104,6 +105,7 @@
     "promptly": "^3.2.0",
     "proxy-agent": "^6.4.0",
     "semver": "^7.6.3",
+    "split2": "^4.2.0",
```

Comment:
Taking a new dependency here to split a stream on new lines. This is fairly complex to do, so rather have it handled by a tested module. The dep is very popular (>14mio weekly downloads) and is maintained by an extermely well-known Node.JS core contributor.

https://www.npmjs.com/package/split2